tasks.c

```diff
@@ -3945,6 +3945,10 @@ static void prvCheckTasksWaitingTermination( void )
          * want to allocate and clean RAM statically. */
         portCLEAN_UP_TCB( pxTCB );
 
+        /* Remove compiler warning about unused variables when portCLEAN_UP_TCB
+         * is not being used. */
+        ( void ) pxTCB;

```

Comment:
```suggestion
#ifndef portCLEAN_UP_TCB
        ( void ) pxTCB;
#endif
```

Reply:
Thank you for the review.

This won't work because portCLEAN_UP_TCB is defined. It's just defined to nothing.

```
        #define portCLEAN_UP_TCB( pxTCB )
```

Perhaps the comment wording could be improved. I'm open to suggestions.

Reply:
yeah thank you for pointing out. I updated the changes here.

Reply:
The updated changes also will not work. `portCLEAN_UP_TCB` is defined to nothing, it is not defined to 0.

Reply:
Does this default definition not take care of this? https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/include/FreeRTOS.h#L314

Reply:
It would, except that `portCLEAN_UP_TCB` is already defined in many ports provided in the repository. Examples:

https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/portable/ARMv8M/non_secure/portable/GCC/ARM_CM33/portmacro.h#L269

https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/portable/IAR/ARM_CM33/non_secure/portmacro.h#L269

I could fix/remove these instances of `portCLEAN_UP_TCB` instead if you prefer.